lib/js/dom.nim

```diff
@@ -1683,6 +1683,7 @@ proc decodeURIComponent*(uri: cstring): cstring {.importc, nodecl.}
 proc encodeURIComponent*(uri: cstring): cstring {.importc, nodecl.}
 proc isFinite*(x: BiggestFloat): bool {.importc, nodecl.}
 proc isNaN*(x: BiggestFloat): bool {.importc, nodecl.}
+  ## see also `math.isNaN`.
```

Comment:
This means it's a breaking change if I ``import dom, math``. Not good.

Reply:
no, because math.isNaN is generic.
works:
```
nim r -b:js --eval:'import dom,math; echo isNaN(NaN)'
```